frontend/tests/unit/components/layout/Card.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { mount } from '@vue/test-utils';
+
+import Card from '@/components/layout/Card.vue';
+
+function createWrapper(options?: { slot?: string }) {
+  return mount(Card, {
+    shallow: true,
+    slots: {
+      default: options?.slot ?? '',
+    },
+  });
+}
+
+describe('Card.vue', () => {
+  it('renders given default slot', () => {
+    const wrapper = createWrapper({ slot: '<span>test</span>' });
+
+    expect(wrapper.html()).toContain('<span>test</span>');
+  });
+
+  it('has round corders', () => {
```

Comment:
Nitpick: I don't like too much testing the design. If something purely visual is changed in the design, we should not have to adapt the tests imo.

Reply:
Okay. I'll remove it. 👍🏾

Reply:
✔️